### PR TITLE
Add nodes

### PR DESCRIPTION
**Pull request by runbtc**

### CODE REVIEW

**Review by HenrikJannsen — APPROVED** [review]
utACK

File: core/src/main/java/bisq/core/user/Preferences.java [hunk 1]
```diff
@@ -126,7 +127,8 @@ public final class Preferences implements PersistedDataHost, BridgeAddressProvid
     ));
     private static final ArrayList<String> XMR_TX_PROOF_SERVICES = new ArrayList<>(Arrays.asList(
             "monero3bec7m26vx6si6qo7q7imlaoz45ot5m2b5z2ppgoooo6jx2rqd.onion", // @emzy
-            "devinxmrwu4jrfq2zmq5kqjpxb44hx7i7didebkwrtvmvygj4uuop2ad.onion" // @devinbileck
+            "devinxmrwu4jrfq2zmq5kqjpxb44hx7i7didebkwrtvmvygj4uuop2ad.onion", // @devinbileck
+            "xmrexplrthytnunr4jasr3vnjc6jo5idsyxzv74a7ep7dy7lwcv2eoyd.onion", // @runbtc
```
Comment (HenrikJannsen) [inline core/src/main/java/bisq/core/user/Preferences.java]: Trailing ','

File: core/src/main/java/bisq/core/user/Preferences.java [hunk 2]
```diff
@@ -139,7 +141,8 @@ public final class Preferences implements PersistedDataHost, BridgeAddressProvid
     private static final ArrayList<String> TX_BROADCAST_SERVICES = new ArrayList<>(Arrays.asList(
             "http://mempoolhqx4isw62xs7abwphsq7ldayuidyx2v2oethdhhj6mlo2r6ad.onion/api/tx",     // @wiz
             "http://mempool4t6mypeemozyterviq3i5de4kpoua65r3qkn5i3kknu5l2cad.onion/api/tx",     // @emzy
-            "http://mempoolcutehjtynu4k4rd746acmssvj2vz4jbz4setb72clbpx2dfqd.onion/api/tx"      // @devinbileck
+            "http://mempoolcutehjtynu4k4rd746acmssvj2vz4jbz4setb72clbpx2dfqd.onion/api/tx",     // @devinbileck
+            "http://runbtcx3wfygbq2wdde6qzjnpyrqn3gvbks7t5jdymmunxttdvvttpyd.onion/api/tx",     // @runbtc
```
Comment (HenrikJannsen) [inline core/src/main/java/bisq/core/user/Preferences.java]: Trailing ','

File: core/src/main/java/bisq/core/provider/ProvidersRepository.java [hunk 3]
```diff
@@ -38,7 +38,8 @@ public class ProvidersRepository {
     public static final List<String> DEFAULT_NODES = Arrays.asList(
             "http://emzypricpidesmyqg2hc6dkwitqzaxrqnpkdg3ae2wef5znncu2ambqd.onion/", // @emzy
             "http://devinpndvdwll4wiqcyq5e7itezmarg7rzicrvf6brzkwxdm374kmmyd.onion/", // @devinbileck
-            "http://ro7nv73awqs3ga2qtqeqawrjpbxwarsazznszvr6whv7tes5ehffopid.onion/" // @alexej996
+            "http://ro7nv73awqs3ga2qtqeqawrjpbxwarsazznszvr6whv7tes5ehffopid.onion/", // @alexej996
+            "http://runbtcpn7gmbj5rgqeyfyvepqokrijem6rbw7o5wgqbguimuoxrmcdyd.onion",  // @runbtc
```
Comment (HenrikJannsen) [inline core/src/main/java/bisq/core/provider/ProvidersRepository.java]: Trailing ','

File: core/src/main/java/bisq/core/btc/nodes/BtcNodes.java [hunk 4]
```diff
@@ -64,7 +64,10 @@ public List<BtcNode> getProvidedBtcNodes() {
                         // Devin Bileck
                         new BtcNode("btc1.bisq.services", "devinbtctu7uctl7hly2juu3thbgeivfnvw3ckj3phy6nyvpnx66yeyd.onion", "172.105.21.216", BtcNode.DEFAULT_PORT, "@devinbileck"),
                         new BtcNode(null, "devinbtcyk643iruzfpaxw3on2jket7rbjmwygm42dmdyub3ietrbmid.onion", null, BtcNode.DEFAULT_PORT, "@devinbileck"),
-                        new BtcNode(null, "devinbtcmwkuitvxl3tfi5of4zau46ymeannkjv6fpnylkgf3q5fa3id.onion", null, BtcNode.DEFAULT_PORT, "@devinbileck")
+                        new BtcNode(null, "devinbtcmwkuitvxl3tfi5of4zau46ymeannkjv6fpnylkgf3q5fa3id.onion", null, BtcNode.DEFAULT_PORT, "@devinbileck"),
+
+                        // runbtc
+                        new BtcNode(null, "runbtcnd22qxdwlmhzsrw6zyfmkivuy5nuqbhasaztekildcxc7lseyd.onion", null, BtcNode.DEFAULT_PORT, "@runbtc"),
```
Comment (HenrikJannsen) [inline core/src/main/java/bisq/core/btc/nodes/BtcNodes.java]: Trailing ','

**Review by HenrikJannsen — DISMISSED** [review]
NACK

**Review by HenrikJannsen — APPROVED** [review]
utACK

**Review by alejandrogarcia83 — APPROVED** [review]
utACK

**Pull request merged**